### DISCUSSION TITLE
Is it possible to add offset to a date?

### DISCUSSION THREAD

**Discussion by timjstacey** [thunderclient/thunder-client-support#839] [Q&A] [answered]
I'd like to create a timestamp using `{{#dateISO}}` with a couple of minutes offset.  I can't see anything specific about date manipulation in the readme and was wondering if it's possible, if so is it documented somewhere?

**Comment by rangav** [comment, marked as answer]
Currently not possible, but it's in roadmap

you can express your interest below 
https://github.com/rangav/thunder-client-support/issues/656

**Comment by rangav** [comment]
This feature is now implemented and published to marketplace, please update to `v1.20.1`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v1.20.1

Please let me know your feedback.

<img width="1304" alt="Screenshot 2022-10-19 at 08 12 16" src="https://user-images.githubusercontent.com/8637550/196635142-a778cb30-25ca-479e-99d8-ada66bf2e66b.png">

**Reply by timjstacey** [reply]
This is awesome, thank you!
